app/views/candidate_mailer/new_offer_made.text.erb

```diff
@@ -10,4 +10,8 @@ Congratulations! You have an offer from <%= @provider_name %> to study <%= @cour
 
 Contact <%= @provider_name %> if you have any questions about this.
 
-[Sign into your account to respond to your offer](<%= candidate_magic_link(@application_form.candidate) %>).
\ No newline at end of file
+<% if (CycleTimetable.decline_by_default_date - 4.weeks).before? Time.zone.now %>
```

Comment:
I'd prefer this to be on the ruby side of the mailer

One part `CycleTimetable.decline_by_default_date - 4.weeks` 
or the whole part `(CycleTimetable.decline_by_default_date - 4.weeks).before? Time.zone.now`

but is a personal preference, feel free to listen or not.

Reply:
Sounds good, I've made the change